lib/mobility-core/src/Kernel/External/Notification/FCM/Flow.hs

```diff
@@ -101,16 +101,17 @@ createApnsPayload androidData isMutable =
       def{fcmAlert = Just fcmAlert,
           fcmData = Just androidData,
           fcmCategory = Just androidData.fcmNotificationType,
-          fcmMutableContent = if isMutable then 1 else 0
+          fcmMutableContent = if isMutable then 1 else 0,
+          fcmSound = Just $ fromMaybe "" androidData.fcmNotificationJSON.fcmdSound
          }
     title :: Maybe FCMNotificationTitle
     title = androidData.fcmNotificationJSON.fcmdTitle
 
     body :: Maybe FCMNotificationBody
     body = androidData.fcmNotificationJSON.fcmdBody
 
-createAndroidNotification :: FCMNotificationTitle -> FCMNotificationBody -> FCMNotificationType -> FCMAndroidNotification
-createAndroidNotification title body notificationType =
+createAndroidNotification :: FCMNotificationTitle -> FCMNotificationBody -> FCMNotificationType -> Maybe Text -> FCMAndroidNotification
+createAndroidNotification title body notificationType sound =
```

Comment:
also we are calling it with nothing everywhere in your NY pr, will we be passing some value for it ?